### ISSUE TITLE
Attribute for HTML Inline should *not* decode as URI

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/markdown-transform#94]
There is no reason why arbitrary HTML inline attributes should be decoded, this is an assumption made only for CiceroMark wrapped variables:

```
bash-3.2$ cat test/data/inline.md
This is a <custom src="property%20or%20something"/> property.
bash-3.2$ ../markdown-cli/index.js parse --sample test/data/inline.md 
12:12:12 - info:
{
  "$class": "org.accordproject.commonmark.Document",
  "xmlns": "http://commonmark.org/xml/1.0",
  "nodes": [
    {
      "$class": "org.accordproject.commonmark.Paragraph",
      "nodes": [
        {
          "$class": "org.accordproject.commonmark.Text",
          "text": "This is a "
        }, 
        {
          "$class": "org.accordproject.commonmark.HtmlInline",
          "tag": {
            "$class": "org.accordproject.commonmark.TagInfo",
            "tagName": "custom",
            "attributeString": "src = "property%20or%20something" ",
            "attributes": [
              {
                "$class": "org.accordproject.commonmark.Attribute",
                "name": "src",
                "value": "property or something"
              }
            ],
            "content": "",
            "closed": true
          },
          "text": "<custom src="property%20or%20something"/>"
        }, 
        {
          "$class": "org.accordproject.commonmark.Text",
          "text": " property."
        }
      ]
    }
  ]
}
```

**Comment by jeromesimeon** [comment]
Proposed fix:
- Do *not* decode attributes in the CommonMarkTranformer
- Move the decode/encode part in the CiceroMarkTransformer

**Comment by jeromesimeon** [comment]
Fixed in #93
